addons/l10n_in_ewaybill_stock/models/l10n_in_ewaybill.py

```diff
@@ -436,6 +436,11 @@ def _generate_ewaybill_direct(self):
         self._l10n_in_ewaybill_stock_handle_zero_distance_alert_if_present(response)
         self._write_successfully_response(response_values)
         self._cr.commit()
+        pdf_content = self.env["ir.actions.report"]._render_qweb_pdf("l10n_in_ewaybill_stock.action_report_ewaybill", res_ids=[self.id])[0]
+        self.message_post(
+            body="E-Way Bill has been generated",
```

Comment:
```suggestion
            body=_("E-Way Bill has been generated"),
```
Always use translator for static strings :) and don't forget to change the `.pot` file ;)